_field-types/supported-field-types/date.md

```diff
@@ -62,7 +62,7 @@ OpenSearch has built-in date formats, but you can also create your own custom fo
 
 ## Default format
 
-As of OpenSearch 2.12, the default date format is `strict_date_time_no_millis||strict_date_optional_time||epoch_millis`. To revert the default format back to `strict_date_optional_time||epoch_millis` (the default format for OpenSearch 2.11 and earlier), set the `opensearch.experimental.optimization.datetime_formatter_caching.enabled` feature flag to `false`. For more information about enabling and disabling feature flags, see [Enabling experimental features]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/experimental/).
+As of OpenSearch 2.12, there is a new experimental default date format `strict_date_time_no_millis||strict_date_optional_time||epoch_millis` (the default format is `strict_date_optional_time||epoch_millis`). To use it, please set the `opensearch.experimental.optimization.datetime_formatter_caching.enabled` feature flag to `true`. For more information about enabling and disabling feature flags, see [Enabling experimental features]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/experimental/).
```

Comment:
A little confused here @reta. Are you saying that as of OpenSearch 2.12, the default has changed from `strict_date_time_no_millis` to `strict_date_optional_time||epoch_millis` as long as the experimental setting is enabled? 

```suggestion
As of OpenSearch 2.12, there is a new experimental default date format `strict_date_time_no_millis||strict_date_optional_time||epoch_millis` (the default format is `strict_date_optional_time||epoch_millis`). To use it,  set the `opensearch.experimental.optimization.datetime_formatter_caching.enabled` feature flag to `true`. For more information about enabling and disabling feature flags, see [Enabling experimental features]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/experimental/).
```

Reply:
> A little confused here @reta. Are you saying that as of OpenSearch 2.12, the default has changed from `strict_date_time_no_millis` to `strict_date_optional_time||epoch_millis` as long as the experimental setting is enabled?

Thanks @Naarcha-AWS , what happened is that we rolled out experimental settings that was set to `true` by default and changed the default datetime format. It caused issues to our users so we are basically correcting that by deactivating this experimental  setting by default, so the default datetime format stays as it was before.

Reply:
So the default format would be `strict_date_time_no_millis||strict_date_optional_time||epoch_millis` when `the `opensearch.experimental.optimization.datetime_formatter_caching.enabled` feature flag is set to `true`?

Reply:
> So the default format would be `strict_date_time_no_millis||strict_date_optional_time||epoch_millis` when the `opensearch.experimental.optimization.datetime_formatter_caching.enabled` feature flag is set to `true`?

That is correct